.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: Test Python package
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+    tags:
+      - 'v*'
+      - '!v*-dev'
+
+jobs:
+  build:
```

Comment:
This should probably be limited to the zeek-client repo as well so forks don't run the workflows. Add `if: github.repository == 'zeek/zeek-client'` here.

Reply:
Ah yeah, good one. I forgot because I was testing the CI in my own fork. Coming up.